.github/workflows/versioning.yml

```
@@ -25,21 +27,36 @@
 
-      - name: Push Changes and Tags
+      - name: Push Changes to Temporary Branch
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          git push https://x-access-token:${GITHUB_TOKEN}@github.com/${{ github.repository }}.git HEAD
+
+      - name: Merge Changes into Main
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:2:33: Double quote to prevent globbing and word splitting

(shellcheck)